Source/cmNinjaNormalTargetGenerator.cxx

```diff
@@ -927,6 +946,25 @@ void cmNinjaNormalTargetGenerator::WriteNvidiaDeviceLinkStatement(
   this->WriteNvidiaDeviceLinkRule(usedResponseFile, config);
 }
 
+void cmNinjaNormalTargetGenerator::CreateLinkCommand(
```

Comment:
Did you test your solution? Please update main.py if there is a need to do it. https://github.com/Software-Analysis-Team/link-commands

Reply:
I tested it